src/validators/entry/index.js

```diff
@@ -1,9 +1,7 @@
-import path from 'path'
 import {
   isString, isArray,
   isObject, map,
 } from 'lodash'
-import {fileExists} from '../../utils'
```

Comment:
I don't think anything else uses this util function, and it's kind of a mess anyway. Care to remove it and its tests? :+1: